backend/src-common/src/main/java/com/siemens/sw360/mail/MailUtil.java

```diff
@@ -0,0 +1,124 @@
+package com.siemens.sw360.mail;
+
+import java.util.*;
+import javax.mail.*;
+import javax.mail.internet.*;
+
+import com.siemens.sw360.datahandler.common.CommonUtils;
+import org.apache.log4j.Logger;
+
+/**
+ * Provides the possiblity to send mail from SW360
+ *
+ * @author birgit.heydenreich@tngtech.com
+ */
+public class MailUtil {
+
+    private static final Logger log = Logger.getLogger(MailUtil.class);
+
+    public static final String MAIL_PROPERTIES_FILE_PATH = "/sw360.properties";
+    private Properties loadedProperties;
+    private Session session;
+
+    private String from;
+    private String host;
+    private String port;
+    private String isAuthenticationNecessary;
+    private String enableStarttls;
+    private String login;
+    private String password;
+    private String enableSsl;
+    private String enableDebug;
+
+    private String subject;
+    private String text;
+
+    public MailUtil() {
+        loadedProperties = CommonUtils.loadProperties(MailUtil.class, MAIL_PROPERTIES_FILE_PATH);
+        setBasicProperties();
+        setSession();
+    }
+
+    private void setBasicProperties() {
+        from = loadedProperties.getProperty("MailUtil_from", "");
+        host = loadedProperties.getProperty("MailUtil_host", "localhost");
```

Comment:
The default host should be `""`. Otherwise it could work by chance or create errors in a service running on port 25